plugin/src/main/scala/com/typesafe/genjavadoc/AST.scala

```diff
@@ -96,7 +96,7 @@ trait AST { this: TransformCake ⇒
   def access(m: Modifiers, topLevel: Boolean): String = {
     if (m.isPublic) "public"
     else if (m.isProtected && !topLevel) "protected"
-    else if (m.isPrivate && !topLevel) "private"
+    else if (m.isPrivate && !topLevel) "" //"private"
```

Comment:
Can it be more specific, like: `if (m.isInterface || m.hasStaticFlag) "" else "private"` ?